keyboards/gkeyboard/greatpad/keymaps/via/keymap.c

```diff
@@ -0,0 +1,93 @@
+    [1] = LAYOUT(
+        KC_TRNS,                                    KC_TRNS,
+        KC_TRNS,  KC_TRNS,  KC_TRNS,    KC_TRNS,    KC_TRNS,
+        KC_TRNS,  KC_TRNS,  KC_TRNS,    KC_TRNS,    KC_TRNS,
+        KC_TRNS,  KC_TRNS,  KC_TRNS,    KC_TRNS,    KC_TRNS,
+        KC_TRNS,  KC_TRNS,  KC_TRNS,    KC_TRNS,    TO(2)
+    ),
+    [2] = LAYOUT(
+        KC_TRNS,                                    KC_TRNS,
+        KC_TRNS,  KC_TRNS,  KC_TRNS,    KC_TRNS,    KC_TRNS,
+        KC_TRNS,  KC_TRNS,  KC_TRNS,    KC_TRNS,    KC_TRNS,
+        KC_TRNS,  KC_TRNS,  KC_TRNS,    KC_TRNS,    KC_TRNS,
+        KC_TRNS,  KC_TRNS,  KC_TRNS,    KC_TRNS,    TO(3)
+    ),
+    [3] = LAYOUT(
+        KC_TRNS,                                    KC_TRNS,
+        KC_TRNS,  KC_TRNS,  KC_TRNS,    KC_TRNS,    KC_TRNS,
+        KC_TRNS,  KC_TRNS,  KC_TRNS,    KC_TRNS,    KC_TRNS,
+        KC_TRNS,  KC_TRNS,  KC_TRNS,    KC_TRNS,    KC_TRNS,
+        KC_TRNS,  KC_TRNS,  KC_TRNS,    KC_TRNS,    TO(4)
+    ),
+    [4] = LAYOUT(
+        KC_TRNS,                                    KC_TRNS,
+        KC_TRNS,  KC_TRNS,  KC_TRNS,    KC_TRNS,    KC_TRNS,
+        KC_TRNS,  KC_TRNS,  KC_TRNS,    KC_TRNS,    KC_TRNS,
+        KC_TRNS,  KC_TRNS,  KC_TRNS,    KC_TRNS,    KC_TRNS,
+        KC_TRNS,  KC_TRNS,  KC_TRNS,    KC_TRNS,    TO(5)
+    ),
+    [5] = LAYOUT(
+        KC_TRNS,                                    KC_TRNS,
+        KC_TRNS,  KC_TRNS,  KC_TRNS,    KC_TRNS,    KC_TRNS,
+        KC_TRNS,  KC_TRNS,  KC_TRNS,    KC_TRNS,    KC_TRNS,
+        KC_TRNS,  KC_TRNS,  KC_TRNS,    KC_TRNS,    KC_TRNS,
+        KC_TRNS,  KC_TRNS,  KC_TRNS,    KC_TRNS,    TO(6)
+    ),
+    [6] = LAYOUT(
+        KC_TRNS,                                    KC_TRNS,
+        KC_TRNS,  KC_TRNS,  KC_TRNS,    KC_TRNS,    KC_TRNS,
+        KC_TRNS,  KC_TRNS,  KC_TRNS,    KC_TRNS,    KC_TRNS,
+        KC_TRNS,  KC_TRNS,  KC_TRNS,    KC_TRNS,    KC_TRNS,
+        KC_TRNS,  KC_TRNS,  KC_TRNS,    KC_TRNS,    TO(7)
+    ),
+    [7] = LAYOUT(
+        KC_TRNS,                                    KC_TRNS,
+        KC_TRNS,  KC_TRNS,  KC_TRNS,    KC_TRNS,    KC_TRNS,
+        KC_TRNS,  KC_TRNS,  KC_TRNS,    KC_TRNS,    KC_TRNS,
+        KC_TRNS,  KC_TRNS,  KC_TRNS,    KC_TRNS,    KC_TRNS,
+        KC_TRNS,  KC_TRNS,  KC_TRNS,    KC_TRNS,    TO(8)
+    ),
+    [8] = LAYOUT(
+        KC_TRNS,                                    KC_TRNS,
+        KC_TRNS,  KC_TRNS,  KC_TRNS,    KC_TRNS,    KC_TRNS,
+        KC_TRNS,  KC_TRNS,  KC_TRNS,    KC_TRNS,    KC_TRNS,
+        KC_TRNS,  KC_TRNS,  KC_TRNS,    KC_TRNS,    KC_TRNS,
+        KC_TRNS,  KC_TRNS,  KC_TRNS,    KC_TRNS,    TO(9)
+    ),
+    [9] = LAYOUT(
+        KC_TRNS,                                    KC_TRNS,
+        KC_TRNS,  KC_TRNS,  KC_TRNS,    KC_TRNS,    KC_TRNS,
+        KC_TRNS,  KC_TRNS,  KC_TRNS,    KC_TRNS,    KC_TRNS,
+        KC_TRNS,  KC_TRNS,  KC_TRNS,    KC_TRNS,    KC_TRNS,
+        KC_TRNS,  KC_TRNS,  KC_TRNS,    KC_TRNS,    TO(0)
```

Comment:
Blank layers should be removed.
```suggestion
        KC_P0,    KC_PDOT,  KC_PENT,    KC_PPLS,    _______
```